libp2p/network/connection/swarm_connection.py

```diff
@@ -19,22 +20,27 @@ class SwarmConn(INetConn):
     muxed_conn: IMuxedConn
     swarm: "Swarm"
     streams: Set[NetStream]
-    event_closed: asyncio.Event
-
-    _tasks: List["asyncio.Future[Any]"]
+    event_started: trio.Event
```

Comment:
Since this is declared on the parent class it seems un-necessary to specify it here too.